docs/data.rst

```diff
@@ -229,15 +226,15 @@ It is possible to obtain the dictoinary of a nested struct using
    for sample in input.valid_data_iterator:
       point = sample.get_dictionary("my_point")
 
-It is also possible to supply "member_name" as an element of an array (if the
+It is also possible to supply ``member_name`` as an element of an array (if the
 type of the array is complex):
 
 .. testcode::
 
    for sample in input.valid_data_iterator:
```

Comment:
See my previous comment before this commit about moving this to the sequences/arrays section